app/src/main/res/values/strings.xml

```diff
@@ -81,11 +81,20 @@
     <string name="mission_null">Cannot launch a null mission</string>
     <string name="back_to_home_button">Back to Home</string>
     <string name="back_to_user_button">Back to User</string>
+
+    <string name="mission_already_in_progress">A mission is already in progress. Wait of it to end or cancel it before starting a new one.</string>
```

Comment:
Typo: "wait for it" instead of "wait of it"

Reply:
This is resolved